apps/src/code-studio/initializeBlockPreview.js

```diff
@@ -8,7 +8,10 @@ function update(blockSpace, container, editor) {
     // catch and ignore all errors.
     var xml = Blockly.Xml.textToDom(editor.getValue());
```

Comment:
Do we need this variable? Could we instead pass `editor.getValue()` into the function call below?

Reply:
Good call!